### PR TITLE
[OC-448]: Fetch contributor on item and collection

### PR DESCRIPTION
**Pull request by philipyoungg**
Add `contributor` on item get endpoint and `contributors` on collections get endpoint.

### CODE REVIEW

File: src/collections/collections.ts [hunk 1]
```diff
@@ -91,9 +91,25 @@ export const getById = async (event: APIGatewayEvent, context: Context): Promise
       query = `
         SELECT
           collections.*,
+          (
+            SELECT json_agg(
+              json_build_object(
+                'id', profile.id,
+                'name', profile.full_name,
+                'isProfilePublic', profile.public_profile
```
Comment (matthewberryman) [inline src/collections/collections.ts:99]: you should check this in the SQL and not pass it back if false

File: src/items/model.ts [hunk 2]
```diff
@@ -164,8 +164,21 @@ export const getItemBy = async (field, value, isAdmin: boolean = false, isContri
       query = `
           SELECT
             item.*,
+            (
+              SELECT json_build_object(
+              'id', profile.id,
+              'name', profile.full_name,
+              'isProfilePublic', profile.public_profile
```
Comment (matthewberryman) [inline src/items/model.ts:171]: ditto

File: src/collections/collections.ts (hunk 1 above)
Comment (philipyoungg) [reply src/collections/collections.ts:99]: I was doing this as we still need to list contributors even though their profile is not public right? 
On front-end, User with `isProfilePublic` is wrapped with link navigation. Non-public one isn't.

**Comment by matthewberryman** [comment]
Oh, right. Yes, you are correct. Sorry I forgot about the context for a second (my brain was still thinking about search)

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by philipyoungg** [comment]
@matthewberryman no worries. Take a rest 😊. This can wait.

**Pull request merged**